### DISCUSSION TITLE
Problem with await

### DISCUSSION THREAD

**Discussion by Alvarobaez10** [adonisjs/core#3125] [Help]
![image](https://user-images.githubusercontent.com/47068171/131581886-75fa8ab1-c2d2-4ae8-9361-444bf2260cbe.png)

I have the following code, i try to create a new register, but it return empty, it doesn't wait for promise

**Comment by sergefabre** [comment]
Hey
Are you try before : 
let arbol = new Arbol()
arbol.fill(request.body())

see with debugger arbol

and after....
Await arbol.save()

(eventually)
await request.validate(ValidateArbol)

**Comment by feniljariwala82** [comment]
Write creation code into try/catch, console error, as per my knowledge it is throwing an error. Try this out.

**Comment by sergefabre** [comment]
the return is after response. 
suddenly, is the return executed?
